File: app/src/main/java/com/devusercode/upchat/models/UserPair.java
package com.devusercode.upchat.models;

public class UserPair {
    private final User user;
    private final String cid;

    public UserPair(User user, String conversationId) {
        this.user = user;
        this.cid = conversationId;
    }

    public User getUser() {
        return user;
    }

    public String getConversationId() {
        return cid;
    }
}

File: core/src/main/java/com/devusercode/core/domain/chat/model/UserPair.kt
package com.devusercode.core.domain.chat.model

import com.devusercode.core.domain.user.model.User

data class UserPair(
    val user: User,
    val conversationId: String,
    val lastMessageText: String? = null,
    val lastMessageTimeEpochMs: Long? = null,
)
